lib/view_component/slotable_v2.rb

```diff
@@ -142,6 +142,19 @@ def renders_many(slot_name, callable = nil)
         register_slot(slot_name, collection: true, callable: callable)
       end
 
+      def renders_one?(slot_name)
+        registered_slots.dig(slot_name, :collection) == false
+      end
+
+      def renders_many?(slot_name)
+        registered_slots.dig(slot_name, :collection) == true
+      end
+
+      def renders_many_item?(slot_name)
```

Comment:
I find this naming a bit confusing compared to the other two... but I'm not sure I have a better idea...

Reply:
I had the same thought. Not great but..